tests/helpers/graphical_units.py

```diff
@@ -0,0 +1,54 @@
+"""Helpers functions for devs to set up new graphical-units data.  
+"""
```

Comment:
unnecessary line break
```suggestion
"""Helpers functions for devs to set up new graphical-units data."""
```